src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -102,6 +102,7 @@ class _SimulationEventNames(NamedTuple):
     AZITHROMYCIN_ACCESS = "azithromycin_access"
     MISOPROSTOL_ACCESS = "misoprostol_access"
     CPAP_ACCESS = "cpap_access"
+    ACS_ACCESS = "acs_access"
```

Comment:
Yea you should definitely update this class to be iterable to make your life easier. I should have done that originally.